Plugin~/WebRTCPlugin/Codec/NvCodec/NvEncoderCuda.h

```diff
@@ -0,0 +1,12 @@
+﻿#pragma once
+#include "NvEncoder.h"
+
+namespace WebRTC {
+    class NvEncoderCuda : public NvEncoder {
+    public:
+        NvEncoderCuda(uint32_t nWidth, uint32_t nHeight, IGraphicsDevice* device);
+        virtual ~NvEncoderCuda() = default;
+    protected:
+        virtual void* AllocateInputResourceV(ITexture2D* tex) override;
+    };
```

Comment:
if we can delete AllocateInputResourceV. class NvEncoderCuda does not need anymore.